.github/CODEOWNERS

```diff
@@ -1,2 +1 @@
-# See go/codeowners - automatically generated for confluentinc/druid:
-*	@confluentinc/observability
+*	@confluentinc/obs-data @confluentinc/obs-oncall
```

Comment:
I don't think we need to include oncall in the reviewers for this repo

Reply:
This is an inherited change